evap/staff/tests.py

```diff
@@ -77,7 +77,7 @@ def test_sample_xls(self):
     def test_sample_user_xls(self):
         page = self.app.get("/staff/user/import", user='user')
 
-        original_user_count = UserProfile.objects.all().count()
+        original_user_count = UserProfile.objects.count()
```

Comment:
if it's not too much work, please separate the unrelated changes into a new commit.